developer-docs/README.md

```diff
@@ -64,102 +64,162 @@ SEMATIC_API_ADDRESS: https://<my_prod1_host:port>
 SEMATIC_API_KEY: <my_prod1_api_key>
 ```
 
-## Releasing
-
-**Note:** Actually pushing the released wheel can only be done if you have
-access to the PyPi repo, which is limited to employees of Sematic.
+## Building the Wheel
 
-We cut releases from the `main` branch, following these steps:
+If you want to build the wheel from source, do:
 
-- Bump the version in `wheel_constants.bzl`, `sematic/versions.py`,
-  and `helm/sematic-server/Chart.yaml`
-- Increment the minor version of the `version` field in
-  `helm/sematic-server/Chart.yaml`. Note that this will be DIFFERENT
-  from the Sematic version you changed in prior steps (chart version
-  will be 1.X.Y, Sematic version is 0.M.N).
-- Update `changelog.md` with the new version number and any missing change
-  entries
-- Make the bump commit
-- Build the UI:
 ```bash
 $ make ui
```

Comment:
Just an FYI, now `make wheel` auto builds `make ui` if ui has never been built before. But it doesn't rebuild UI if anything changed, so relying on this auto behavior is not safe. 

Maybe consider adding a comment to explain, or feel free to skip it if it's not that important.